data/json/snippets/e_logs.json

```diff
@@ -0,0 +1,25 @@
+      },
+      {
+        "id": "local_files_simple_2",
+        "text": "system/storage/Camera/IMG_202X0401_111567.jpg  \n\nHigh quality photo of an angry civil mob that suppressed the line of shielded policemans.  There is few dead people on the edge of a photo, one of which lay with chopped head."
+      },
+      {
+        "id": "local_files_simple_3",
+        "text": "apps/myMail  \n\nby:dnance@mymail.com  \nto:All  \nSubject: Tragic Loss of Rob Dixon  \n\nDear Colleagues,  \n\nIt is with a heavy heart that I have to inform you all of the passing of the head of supply chain department, Rob Dixon.  He was at the office late last night trying to ensure that everything was under control when he was caught in one of the riots that have been happening all over the city.  Despite all efforts, Rob did not make it.  \nRob was more than just a boss to us, he was a friend and mentor to many of us.  Rob always put the company and its employees first and he will be deeply missed.  \nPlease keep Rob’s family and loved ones in your thoughts and prayers during this difficult time.  The company is working closely with the authorities to ensure that Rob’s death is investigated fully.  \n\nIn the meantime, all of Rob's responsibilities will be temporarily handled by James Sullivan, his senior assistant.  If you have any urgent matters, please reach out to James.  \nWe will provide updates as they become available and further details on arrangements to celebrate Rob's life.  \n\nSincerely,\nDahila Nance\nHead of HR department."
+      },
+      {
+        "id": "local_files_simple_4",
+        "text": "apps/Harmony \n\nlegendary_steak: Dude, just met the most fucking terrifying thing in my life.  I was on my way home and I saw this deer on the side of the road and I had to do a double take because it had EIGHT legs! \n\nW. F. Gibson (real): bro are you high? \n\nlegendary_steak: I'm serious! \n\nlegendary_steak: I almost piss my pants! \n\nlegendary_steak: There was also something wrong with their eyes, but i didn't focus \n\nW. F. Gibson (real): Did you take a picture or something? \n\nlegendary_steak: I think i got it on my dashcam, but i don't think it worked good - i stepped on a gas with full force because it started to approach me, almost killed myself \n\nW. F. Gibson (real): Mad things happened in the last few weeks, yeah \n\nlegendary_steak: Yeah, it was bad.  I've been hearing all these stories about weird things happening and now I've seen it for myself \n\nW. F. Gibson (real): Maybe i need to buy shotgun or two."
```

Comment:
```suggestion
        "text": "apps/Harmony \n\nlegendary_steak: Dude, just met the most fucking terrifying thing in my life.  I was on my way home and I saw this deer on the side of the road and I had to do a double take because it had EIGHT legs! \n\nW. F. Gibson (real): bro are you high? \n\nlegendary_steak: I'm serious! \n\nlegendary_steak: I almost pissed my pants! \n\nlegendary_steak: There was also something wrong with the eyes, but I didn't focus \n\nW. F. Gibson (real): Did you take a picture or something? \n\nlegendary_steak: I think I got it on my dashcam, but I don't think it worked well - I stepped on a gas with full force because it started to approach me, almost killed myself \n\nW. F. Gibson (real): Mad things happened in the last few weeks, yeah \n\nlegendary_steak: Yeah, it was bad.  I've been hearing all these stories about weird things happening and now I've seen it for myself \n\nW. F. Gibson (real): Maybe I need to buy a shotgun or two."
```